rollup.config.mjs

```diff
@@ -1,4 +1,7 @@
-import pkg from './package.json';
+import { createRequire } from 'node:module';
+const require = createRequire(import.meta.url);
+
+const pkg = require('./package.json');
```

Comment:
I trust your change, but I've never seen `createRequire` before. Why do we need to do it this way? Is this the recommended standard?

Reply:
It is - since we're using native ESM here, we can't just `import package.json`, as it's not a proper ES Module. Per the Rollup Documentation: https://rollupjs.org/guide/en/#importing-packagejson, we take this Node-native approach to getting the data

Reply:
Gotcha. Thanks for the clarification!